scipy/stats/_distribution_infrastructure.py

```diff
@@ -4365,9 +4385,6 @@ def wrapped(self, p, *args, loc, scale, sign, **kwargs):
 
 class TransformedDistribution(ContinuousDistribution):
```

Comment:
Hmmm this makes a discrete distribution continuous which clearly isn't right!

Reply:
```suggestion
class TransformedDistribution(UnivariateDistribution):
```
?

Reply:
I'll double check but I think I tried this but it broke the tests for continuous transformed distributions as they no longer have a _logpxf_dispatch or the other methods defined by ContinuousDistribution